content/projects/katas/level-1/_index.md

```diff
@@ -57,7 +57,16 @@ In other words, there should be no directories. Only modules.
 
 If you are writing JS code then your files would follow a similar pattern. You would just have `.js` instead of `.py` files.
 
-## Note to Javascript developers:
+If you are using Java, your file names should be capitalized, like so:
```

Comment:
I think you can also add a piece that the Directory structure is also similar to the python and javascript

Reply:
There is a line that says that on line 58, it reads:
"""
If you are writing JS code then your files would follow a similar pattern. You would just have `.js` instead of `.py` files.
"""

Reply:
yeah to me that only mentions javaScript and Python  but not java